MVVM Module/Source/Feed UI/Controllers/FeedViewController.swift

```diff
@@ -5,6 +5,8 @@
 import UIKit
 
 public final class FeedViewController: UITableViewController, UITableViewDataSourcePrefetching {
+	@IBOutlet weak public var errorView: ErrorView?
```

Comment:
This property doesn't need to be `weak`. Also, the setter can be private to prevent clients from changing the instance:

```suggestion
	@IBOutlet private(set) public var errorView: ErrorView!
```